cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/reserve_transfer.rs

```diff
@@ -159,19 +263,58 @@ fn reserve_transfer_native_asset_from_system_para_to_relay_fails() {
 	});
 }
 
+/// Reserve Transfers of native asset from Relay to Parachain should work
```

Comment:
it looks like that this test has nothing to do with AssetHub, so it should be placed elsewhere, e.g.:
`/tests/assets/westend/src/tests/reserve_transfer.rs`
or
`/tests/westend/src/tests/reserve_transfer.rs`
or
`/tests/relays/westend/src/tests/reserve_transfer.rs`
or
elsewhere?

Reply:
You have a point, but I don't see a good alternative - we have a lot of tests in both `assets` and `bridges` involving the relay chains.. pure `relay` tests are very few (right now it'd just be this one discussed here).

So I'm keeping it under `assets` - it's not an `AssetHub` test, yes, but still it is an `assets` test if you ask me..

Also `cumulus/parachains/integration-tests/emulated` already has toooo many targets and crates to build if you ask me, I really don't want to add more unless really necessary/useful.

Reply:
This whole split of tests based on chains is not the best for integration tests that by definition use more than one chain.. we'll keep having these types of discussions, unless we start structuring them by topic/scenario, rather than chains involved.

Reply:
Yeah, I agree, we should restructure them - there is an older @NachoPal 's issue: https://github.com/paritytech/polkadot-sdk/issues/1389.
There are also various tests for AssetHubs that are not directly related to 'assets,' such as setting the XCM version or opening an HRMP channel. Some of these tests are not even related to the AssetHub itself, for example, there are tests using only the relay chain and penpal.